messages/run.js

```diff
@@ -30,7 +30,6 @@ module.exports = {
 		'pmdConfigDescriptionLong': 'Location of PMD rule reference XML file to customize rule selection'
 	},
 	"validations": {
-		"mustTargetSomething": "Please specify a codebase using --target.", // TODO: Once --org is implemented, rewrite this message.
```

Comment:
Removing some unused messages, this was a holdover from the `org` flag. Corresponding changes in run.ts.